src/main/java/com/spotify/confidence/ConfidenceFeatureProvider.java

```diff
@@ -171,6 +178,9 @@ public ProviderEvaluation<Value> getObjectEvaluation(
       final ResolvedFlag resolvedFlag = resolveFlagResponse.getResolvedFlags(0);
 
       if (resolvedFlag.getVariant().isEmpty()) {
+        log.info(
+            "The server returned no assignment for the flag. Typically, this happens "
+                + "if no configured rules matches the given evaluation context.");
```

Comment:
This might not be very useful without information of which flag is considered. I also wonder if we should avoid `info` logs and avoid polluting our user's logs when things are functioning, rather keep this extra information under `debug`

Reply:
I updated this to include the flag name and moved it to `debug`.